readme.md

```diff
@@ -1,17 +1,27 @@
-## Java Parser and Abstract Syntax Tree
-
-This package contains a Java 1.0 - Java 9 Parser with AST generation and visitor support.
-
-Our main site is at [JavaParser.org](http://javaparser.org)
+# JavaParser
 
 [![Maven Central](https://img.shields.io/maven-central/v/com.github.javaparser/javaparser-core.svg)](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.javaparser%22%20AND%20a%3A%22javaparser-core%22)
 [![Build Status](https://travis-ci.org/javaparser/javaparser.svg?branch=master)](https://travis-ci.org/javaparser/javaparser)
 [![Coverage Status](https://coveralls.io/repos/javaparser/javaparser/badge.svg?branch=master&service=github)](https://coveralls.io/github/javaparser/javaparser?branch=master)
 [![Join the chat at https://gitter.im/javaparser/javaparser](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/javaparser/javaparser?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## Dependency Management
+This project contains a set of libraries implementing a Java 1.0 - Java 9 Parser with AST generation and advanced features
+like:
+
+* comment attribution
+* lexical preservation
+* code generation

```

Comment:
I'd be fine with moving this kind of descriptive content to javaparser.org.

Reply:
I will remove the list but my problem is that seeing that JavaParser is just a parser could be reductive